wagtail/admin/templates/wagtailadmin/tables/status_tag_cell.html

```diff
@@ -1,3 +1,4 @@
+{% load wagtailadmin_tags %}
 <td {% if column.classname %}class="{{ column.classname }}"{% endif %}>
-    <div class="status-tag {% if primary %}primary{% endif %}">{{ value }}</div>
+    {% if primary %}{% status value classname="w-block primary" %}{% else %}{% status value classname="w-block" %}{% endif%}
```

Comment:
@salty-ivy  there were a few leftover `div` usage items so I have converted them to the status tag also, not sure if these were there already but we can emulate a div by `span` with `w-block` override in the class.

Reply:
Note. I think I'm wrong here. The inline-block on the base styles would be all we need.